hub/core/augment/augment.py

```diff
@@ -0,0 +1,48 @@
+from hub.core.dataset import Dataset
+from typing import List
+from hub.core.transform.transform import ComputeFunction 
+from torch.utils.data.dataloader import DataLoader
+import torch
+import numpy as np
+import time
+
+class Augmenter():    
+  def __init__(self, transformation_dict=None):
+    """
+    Creates an Augmenter object. 
+    
```

Comment:
add more examples how to use it so this would be added into docs. 
and btw add to the docs.